guides/common/modules/ref_troubleshooting-discovery.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Troubleshooting_Discovery_{context}"]
-= Troubleshooting Discovery
+= Troubleshooting discovery
```

Comment:
```suggestion
= Troubleshooting Discovery
```

I think this refers to "Foreman Discovery", a plug-in.

Reply:
Looks like it refers to PXE discovery:

> Check the PXELinux template in use and determine the PXE discovery snippet it includes